examples/Cookbook.ipynb

```diff
@@ -716,33 +880,143 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Create a transaction to upgrade an existing smart contract:"
+    ":::tip\n",
+    "When creating transactions using `SmartContractTransactionsFactory`, even if the ABI is available and provided,\n",
+    "you can still use _typed value_ objects as arguments for deployments and interactions.\n",
+    "\n",
+    "Even further, you can use a mix of _typed value_ objects and plain Python values and objects. For example:\n",
+    "```\n",
+    "args = [U32Value(42), \"hello\", { \"foo\": \"bar\" }, TokenIdentifierValue(\"TEST-abcdef\")];\n",
+    "```\n",
+    ":::"
```

Comment:
this is not properly rendered.

Reply:
looks good in the docs but not on the notebook.
![image](https://github.com/multiversx/mx-sdk-py/assets/73281061/95e71605-c80d-4f16-8297-29bad53b30ee)

Reply:
Removed. We'll separately maintain these markers in mx-docs.